src/common/services/api/designations.service.js

```diff
@@ -253,6 +253,10 @@ class DesignationsService {
           // Map giving links
           if (data.data['jcr:content'].givingLinks) {
             angular.forEach(data.data['jcr:content'].givingLinks, (v, k) => {
+              if (!v || !v.name || !v.url) {
+                return
```

Comment:
Should this instead be `continue`? How will this work if the first item is invalid, but the second is valid?

Reply:
If it were a `for` loop, we'd want to use `continue`. But since it's a `forEach` callback, we can return early and `angular.forEach` will call our callback again with the next item.